roomranger/frontend/src/App.css

```diff
@@ -11,27 +11,74 @@
 
 .footer {
  position: relative;
- /* position:absolute; */
-left:0;
-bottom:0;
-right:0;
+ left:0;
+ bottom: 0;
+ right:0;
  width:100%;
  height: 50px;
- /* background-color: rgb(236, 229, 229); */
+ /*background-color: rgb(236, 229, 229);*/
  text-align: center;
  color: white;
 }
 
 .nbar{
```

Comment:
i think you should command this line too.

Reply:
line 24

Reply:
Done